test/benchee/benchmark/runner_test.exs

```diff
@@ -0,0 +1,217 @@
+defmodule Benchee.Benchmark.RunnerTest do
+  use ExUnit.Case, async: true
+  import Benchee.TestHelpers
+  alias Benchee.Suite
+  alias Benchee.Benchmark
+  alias Benchee.Configuration
+  alias Benchee.Statistics
+  alias Benchee.Test.FakeBenchmarkPrinter, as: TestPrinter
+
+  @config %Configuration{parallel: 1,
+                         time:     40_000,
+                         warmup:   20_000,
+                         inputs:   nil,
+                         print:    %{fast_warning: false, configuration: true}}
+  @system %{elixir: "1.4.0", erlang: "19.1"}
+  @default_suite %Suite{configuration: @config, system: @system}
+
+  defp test_suite(suite_override \\ %{}) do
+    DeepMerge.deep_merge(@default_suite, suite_override)
+  end
+
+  defp run_times_for(suite, job_name, input_name \\ Benchmark.no_input()) do
+    filter_fun = fn(scenario) ->
+      scenario.job_name == job_name && scenario.input_name == input_name
+    end
+    map_fun = fn(scenario) -> scenario.run_times end
+
+    suite.scenarios
+    |> Enum.filter_map(filter_fun, map_fun)
+    |> List.flatten()
+  end
```

Comment:
This is something we eventually wanna provide on `Suite`, one of the downsides of the new approach. I guess this might become more relevant when working on bringing the plugins up to date which I sort of dread :D